pkg/kubectl/describe.go

```diff
@@ -1588,6 +1588,9 @@ func (dd *DeploymentDescriber) Describe(namespace, name string) (string, error)
 		fmt.Fprintf(out, "Labels:\t%s\n", labels.FormatLabels(d.Labels))
 		fmt.Fprintf(out, "Selector:\t%s\n", labels.FormatLabels(d.Spec.Selector))
 		fmt.Fprintf(out, "Replicas:\t%d updated / %d total\n", d.Status.UpdatedReplicas, d.Spec.Replicas)
+		if conditionsLen := len(d.Status.Conditions); conditionsLen > 0 {
+			fmt.Fprintf(out, "Status:\t%s\n", d.Status.Conditions[conditionsLen-1].Type)
```

Comment:
How is Condition{Progressing, false} any different from Condition{Failed, true}? Do we really need to indicate if the condition is true or false?